modules/creating-instance-aws-load-balancer-controller.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.adoc
+
+:_content-type: PROCEDURE
+[id="nw-creating-instance-aws-load-balancer-controller_{context}"]
+= Creating an instance of the AWS Load Balancer Controller using ALB Operator
+
+After installing the operator, you can create an instance of the `AWSLoadBalancerController`. You can install only a single instance of the `aws-load-balancer-controller` in a cluster. The operator reconciles only the resource with the name `cluster`.
+
+.Prerequisites
+
+.Procedure
+
+. Create a `AWSLoadBalancerController` resource:
++
+[source,yaml]
+----
+apiVersion: networking.olm.openshift.io/v1alpha1
+kind: AWSLoadBalancerController <1>
+metadata:
+  name: cluster <2>
+spec:
+  subnetTagging: Auto <3>
+  additionalResourceTags: <4>
+    example.org/cost-center: 5113232
+    example.org/security-scope: staging
+  ingressClass: cloud <5>
+  config:
+    replicas: 2 <6>
+  enabledAddons: <7>
+    - AWSWAFv2 <8>
+----
+
+<1> Defines the `AWSLoadBalancerControlller` resource.
+<2> Deines the `aws-load-balancer-controller` instance name. This instance name gets added as a suffix to all related resources.
+<3> It can define two options such as `Auto` and `Manual`. When the value is set to `Auto` the operator attempts to determine the subnets which belong to the cluster and tags them appropriately. The operator cannot determine the role correctly if the internal subnet tags are not present on internal subnet. So if you installed your cluster on User-Provisioned Infrastructure, you must manually tag the subnets with the appropriate role tags and set the subnet tagging policy to `Manual`.
+<4> Defines the tags used by the controller when it provisions AWS resources.
+<5> The default value for this field is `alb`. The operator provisions an `IngressClass` with the same name if it does not exist.
+<6> Specifies the number of replicas of the controller.
+<7> Specifies add-ons for Ingress resources, which gets specified through annotations.
+<8> Enables the `alb.ingress.kubernetes.io/wafv2-acl-arn` annotation.
+
+. After the AWS Load Balancer Controller is running, create a `deployment`:
```

Comment:
Should we say that the below steps are for the validating purpose?

Reply:
Then we must update the heading as `.Verification steps`.  But I think we are still `creating` instead of `checking or verifying` so IMO we should keep it as it is.   But please let me know if I am missing something.

Reply:
I think `deployment` can be deployment here per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references